molgenis-security/src/main/resources/templates/view-2fa-activation-modal.ftl

```diff
@@ -60,14 +60,14 @@
                                            name="verificationCode"
                                            required/>
                                     <script>
-                                        $('#verification-code-field').pincodeInput({
+                                        var pincodeInput = $('#verification-code-field').pincodeInput({
                                             inputs: 6,
                                             hidedigits: false,
-                                            complete: function (value, e) {
+                                            complete: function () {
                                                 $('#activation-form').submit()
                                             }
-                                        })
-                                        $('input.pincode-input-text.first').attr('autofocus', 'autofocus')
+                                        }).data('plugin_pincodeInput')
+                                        setTimeout(pincodeInput.focus)
```

Comment:
The setTimeout makes sure the focus method is called after this script finishes, hopefully when the input field has been created. I've tested on two browsers and so far with great succes :)

Reply:
Updated the timeout to 1 second